### PR TITLE
Add scenario-based tests

### PR DESCRIPTION
**Pull request by swaroop-sridhar**
Modified Xunit runner to run Scenario Benchmarks
Updating xunit-performance packages

This PR picks out the script and packaging changes from #10536, since these changes are required for #10947

### CODE REVIEW

**Comment by swaroop-sridhar** [comment]
@jorive please review

**Comment by swaroop-sridhar** [comment]
@dotnet-bot  test Ubuntu arm Cross Release Build

File: tests/scripts/run-xunit-perf.cmd [hunk 1]
```diff
@@ -94,11 +94,20 @@ if DEFINED TEST_ENV (
...
+  corerun.exe %CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT% --perf:runid Perf > %BENCHNAME%.out
+) else (
+  corerun.exe PerfHarness.dll %CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT% --perf:runid Perf > %BENCHNAME%.out
+)
 
 @rem optionally generate results for benchview
 if not [%BENCHVIEW_PATH%] == [] (
-  py "%BENCHVIEW_PATH%\measurement.py" xunit "Perf-%BENCHNAME%.xml" --better desc --drop-first-value --append
+  if [%IS_SCENARIO_TEST%] == [1] (
+    py "%BENCHVIEW_PATH%\measurement.py" perftest "Perf-%BENCHNAME%.xml" --better desc --drop-first-value --append
```
Comment (jorive) [inline tests/scripts/run-xunit-perf.cmd]: The `perftest` verb should be replaced with `xunitscenario`

File: tests/scripts/run-xunit-perf.cmd [hunk 2]
```diff
@@ -94,11 +94,20 @@ if DEFINED TEST_ENV (
...
+    echo "SCENARIOTEST"
```
Comment (jorive) [inline tests/scripts/run-xunit-perf.cmd]: Could you please remove the `echo "SCENARIOTEST"` line?

**Comment by swaroop-sridhar** [comment]
Thanks @jorive . I've made the two changes.

**Comment by jorive** [comment]
LGTM!

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu arm Cross Release Build

**Pull request merged**